Arrowgene.Ddon.GameServer/Characters/JobManager.cs

```diff
@@ -509,38 +518,19 @@ public void RemoveAbility(IDatabase database, CharacterCommon character, byte sl
         {
             // TODO: Performance
             List<Ability> equippedAbilities = character.EquippedAbilitiesDictionary[character.Job];
-            lock(equippedAbilities)
+            lock (equippedAbilities)
```

Comment:
I'm not sure if we need this lock. I left it here because the original code had it but I am unsure if it is actually required. While testing before creating the PR I noticed that an exception was raised while unequipping the augments. Seems like it was an outstanding issue but should be fixed now.